frontend/src/Dashboard/CrossingListPage/CrossingListHeader/CrossingListHeader.scss

```diff
@@ -44,40 +44,40 @@
   line-height: 45px;
 }
 
-.CrossingListSortToggle.fullsize {
-  flex: auto;
-  color: white;
-}
-
 .CrossingListSortToggleText.fullsize {
   line-height: 65px;
   text-align: center;
   margin-left: -30%;
   font-size: 1.3em;
 }
 
-.CrossingListSortToggle.smallsize {
+.CrossingListSortToggle {
   flex: auto;
   color: white;
+  cursor: pointer;
+}
+
+.CrossingListSortToggle.smallsize {
+  flex: 1;
 }
 
 .CrossingListSortToggleText.smallsize {
   line-height: 65px;
   text-align: center;
-  margin-left: -30%;
   font-size: 1.3em;
 }
 
 .CrossingListFilterToggle.smallsize {
-  flex: auto;
-  color: white;
-}
-
-.CrossingListFilterToggle.smallsize {
   line-height: 65px;
   text-align: center;
-  margin-left: -30%;
   font-size: 1.3em;
+  cursor: pointer;
+  flex: 1;
+  color: white;
+}
+
+.CrossingListFilterToggle.smallsize.selected {
```

Comment:
we talked IRL about CSS approach. I know that pending and we have some ideas. Made this issue so that discussion can continue: https://github.com/cityofaustin/ctxfloods/issues/59